libraries/rush-lib/src/logic/operations/OperationMetadataManager.ts

```diff
@@ -41,7 +57,9 @@ export class OperationMetadataManager {
   private _metadataFolder: string;
   private _logPath: string;
   private _errorLogPath: string;
+  private _logChunksPath: string;
   private _relativeLogPath: string;
+  private _relativeLogChunksPath: string;
   private _relativeErrorLogPath: string;
```

Comment:
Can all of these be readonly?

Reply:
Updated in [e8b979c](https://github.com/microsoft/rushstack/pull/4668/commits/e8b979ccdd5f37686ce3b576c35bdc0511c7544d)